src/config.rs

```diff
@@ -151,13 +151,10 @@ impl Config {
                 });
             }
         };
-        Config::read_by_path(root.clone(), root).await
+        Config::read_by_path(root).await
     }
 
-    pub async fn read_by_path(
-        path: camino::Utf8PathBuf,
-        root: camino::Utf8PathBuf,
-    ) -> fpm::Result<Config> {
+    pub async fn read_by_path(path: camino::Utf8PathBuf) -> fpm::Result<Config> {
```

Comment:
Nopes. This function must not exist. There is every only one Config object that is only and only constructed by main(). Config is the configuration of the whole FPM program. 

Translations are packages. So we must only read `fpm::Package` for translation. If we need more data in it we will add it to `fpm::Package`. If one common `fpm::Package` is not good enough for current package and translation or dependency, we will create other types.